octobot_tentacles_manager/configuration/tentacles_setup_configuration.py

```diff
@@ -0,0 +1,142 @@
+#  Drakkar-Software OctoBot-Tentacles-Manager
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from copy import copy
+from os.path import join
+
+from octobot_commons.logging.logging_util import get_logger
+from octobot_tentacles_manager.configuration.config_file import read_config, write_config
+from octobot_tentacles_manager.constants import USER_TENTACLE_CONFIG_FILE_PATH, DEFAULT_TENTACLE_CONFIG, \
+    ACTIVATABLE_TENTACLES, DEFAULT_BOT_PATH
+from octobot_tentacles_manager.loaders.tentacle_loading import get_tentacle_classes
+
+
+class TentaclesSetupConfiguration:
+    TENTACLE_ACTIVATION_KEY = "tentacle_activation"
+
+    def __init__(self, bot_installation_path=DEFAULT_BOT_PATH, config_path=USER_TENTACLE_CONFIG_FILE_PATH):
+        self.logger = get_logger(self.__class__.__name__)
+        self.config_path = join(bot_installation_path, config_path)
+        self.tentacles_activation = {}
+
+        # TODO: add user tentacle packages (previously stored in config.json as an URL/path list)
```

Comment:
Are you going to do this in the current PR ?

Reply:
I don't think so, we need to figure out how to handle these first